gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/types/permanent_redirect.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Aws
+  module S3
+    module Types
+      # This error is not modeled.
```

Comment:
What happens if it ever is modeled?  

Would it work to instead add the error w/ the correct members to the API (if none exist) during code generation? (ie: transform the model rather than add custom code)?

Reply:
Yeah I thought about transforming the model. I would prefer to do that, but I don't know all of the cases where PermanentRedirect would be thrown.